OrbitQt/CallTreeWidget.h

```diff
@@ -92,13 +95,21 @@ class CallTreeWidget : public QWidget {
   void SetCallTreeView(std::unique_ptr<CallTreeView> call_tree_view,
                        std::unique_ptr<QIdentityProxyModel> hide_values_proxy_model);
 
+  void ResizeColumnsIfNecessary();
+
   std::unique_ptr<Ui::CallTreeWidget> ui_;
   OrbitApp* app_ = nullptr;
   std::unique_ptr<CallTreeViewItemModel> model_;
   std::unique_ptr<QIdentityProxyModel> hide_values_proxy_model_;
   std::unique_ptr<HighlightCustomFilterSortFilterProxyModel> search_proxy_model_;
   std::unique_ptr<HookedIdentityProxyModel> hooked_proxy_model_;
-  bool columns_already_resized_ = false;
+
+  enum class ColumnResizingState {
+    kInitial = 0,        // CallTreeWidget hasn't got its first size set yet
+    kWidgetSizeSet = 1,  // CallTreeWidget has got its first size, resize the columns when possible
```

Comment:
is resizeEvent something called only once?

Reply:
No, it's called every time the widget is resized (including when the first size is set when the widget is first shown). Have you noticed anything that could be wrong based on this?

Reply:
and we do not want to resize columns if this was not the first one, correct?

Reply:
Exactly, we want to resize only once just to give a reasonable width. At least that's my intention.